nimare/decode/continuous.py

```diff
@@ -238,13 +238,20 @@ class CorrelationDistributionDecoder(Decoder):
     evaluate results based on significance.
     """
 
+    @use_memmap(LGR, n_files=1)
```

Comment:
I don't think we can know how many memmaps are necessary, since there should probably be one file for each feature.

Reply:
Dropping use_memmap from this class.